pkg/controller/elasticsearch/sset/list_test.go

```diff
@@ -79,14 +79,20 @@ func TestStatefulSetList_GetExistingPods(t *testing.T) {
 			},
 		},
 	}
-	client := k8s.WrapClient(fake.NewFakeClient(&pod1, &pod2))
+	// pod not belonging to the sset
+	podNotInSset := corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "pod-not-in-sset",
+			Labels: map[string]string{
+				label.StatefulSetNameLabelName: "not-a-sset",
```

Comment:
maybe more `"different-sset"`

Reply:
agreed, I guess I was too sleepy when I wrote this :stuck_out_tongue_closed_eyes: